mobx/test/store_test.dart

```diff
@@ -6,8 +6,5 @@ class TestStore with Store {}
 void main() {
```

Comment:
One thing I might suggest is getting rid of this file entirely. This was the only test, right? We can add the file back when required.

Reply:
Yes agree, let's delete this test file